src/panels/lovelace/types.ts

```diff
@@ -44,6 +44,7 @@ export interface LovelaceCard extends HTMLElement {
   isPanel?: boolean;
   editMode?: boolean;
   getCardSize(): number | Promise<number>;
+  getSize?(): [number, number];
```

Comment:
We need to think about these names 😅  confusing... first should now be `getHeight` or something...

Reply:
let's specify the size for what it is. Like GridSize. 

Also, should this also be able to be a promise or not?

Reply:
I would try to avoid having a card defining it's size async because it will shift the grid during the loading.

Reply:
It will need to be asynchronous for a card that wraps another custom card, as that card can be lazy loaded